pkg/noderesourcetopology/filter.go

```diff
@@ -68,22 +68,28 @@ func resourcesAvailableInAnyNUMANodes(logKey string, numaNodes NUMANodeList, res
 
 	// Node() != nil already verified in Filter(), which is the only public entry point
 	nodeName := nodeInfo.Node().Name
+	nodeResources := util.ResourceList(nodeInfo.Allocatable)
 
 	for resource, quantity := range resources {
+		if quantity.IsZero() {
+			// why bother? everything's fine from the perspective of this resource
+			klog.V(4).InfoS("ignoring zero-qty resource request", "logKey", logKey, "node", nodeName, "resource", resource)
+			continue
+		}
+
+		if !isNodeSuitable(nodeResources, resource, quantity) {
```

Comment:
Sounds like we're duplicating the regular check which is already covered in the in-tree [NodeResourcesFit](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/noderesources/fit.go) plugin? I'd suggest to define a list of resources (via pluginArgs or by default) so NRT plugin will check those resources only?

Reply:
Indeed @Huang-Wei , this is something I'm investigating and I'll clarify ASAP. Thanks for highlighting this!

Reply:
SGTM.